Setup Assistant stores wrong formated doubles for group_states in the .srdf

When defining a joint state for a group_state in decimal notation setup assistant stores it as:

```
<group_state name="Deposit" group="Arm">     
    <joint name="hand" value="0" />
    <joint name="prisma1" value="0,4449"/>
    <joint name="xaxis" value="0" />
    <joint name="yaxis" value="2,15" />
</group_state>
```

But seems the parser throws an error when loading the robot model, notice the ','. This particular pose appears available, but not work, and not give any error message when trying to define it.

```
[ERROR] [1374223441.626850398]: Unable to parse joint value '0,4449'
[ERROR] [1374223441.626921239]: Unable to parse joint value '2,15'
```

So I had to manually change on this joints to all ',' for '.' :

```
<group_state name="Deposit" group="Arm">     
    <joint name="hand" value="0" />
    <joint name="prisma1" value="0.4449"/>
    <joint name="xaxis" value="0" />
    <joint name="yaxis" value="2.15" />
</group_state> 
```

and with that, no error is prompt when the robot model is loaded.

So probably just need to check that the values are saved with the correct '.' always in any situation. Even in the GUI always are points. So not understand why writes ','.

I'm using groovy desktop from ubuntu 12.04 repository with a custom robot. We thought it could be our locale system settings that mess when the assistant writes the file. But I changed to UK local and still saving comas. No one else noticed?

--- a/src/setup_assistant_main.cpp
+++ b/src/setup_assistant_main.cpp
@@ -40,6 +40,7 @@
 #include <QMessageBox>
 #include <boost/program_options.hpp>
 #include <signal.h>
+#include <locale.h>
 
 static void siginthandler(int param)
 {
@@ -81,6 +82,8 @@ int main(int argc, char **argv)
 
   // Create Qt Application
   QApplication qtApp(argc, argv);
+  // numeric values should always be POSIX
+  setlocale(LC_NUMERIC, "C");
 
   // Load Qt Widget
   moveit_setup_assistant::SetupAssistantWidget saw( NULL, vm );

